### PR TITLE
Improve the documents.

### PR DESCRIPTION
**Pull request by teweitsai**

### CODE REVIEW

File: doc/error-handling/error-handling.rst [hunk 1]
```diff
@@ -320,3 +321,24 @@ If the error was **Excessive Comm Current (code 6066)**, this means the telemetr
...
+Actually, in the normal telescope survey, the M2 should stay at the origin with the minimum stress.
+However, in some cases, we would need to adjust the mirror's position and this might trigger the error to transition the system to leave the closed-loop control to protect the mirror.
+
+If the error code is 6055, you can check the :ref:`lsst.ts.m2gui-user_detailed_force` to see which actuator has the force higher than the threshold of closed-loop control.
+Usually, the axial actuator is related to the z-direction and the tangent link is related to the x- and y-direction.
+Once the actuator is identified, you can move it in steps to have the force lower than the threshold of closed-loop control, and then, put the system into the closed-loop control to recover the system.
+It is noted that the single actuator movement will increase the stress of the system.
+Therefore, the actuator's force should not differ from the threshold too much before transitioning to the closed-loop control.
+Otherwise, the system might be damaged from the high stress.
```
Comment (gabrirod86) [inline doc/error-handling/error-handling.rst:339]: I would add also to check the 4 extra conditions on the tangent links.

File: doc/user-guide/user-guide.rst [hunk 2]
```diff
@@ -86,13 +91,27 @@ Local Mode
...
+By default, the **Diagonostic** state does not have the powers of communication and motor.
+It would be good that you can check the interlock first before transitioning to the **Enabled** state.
 
 Based on the circuit configuration, the interlock might be released after the motor power is on or the motor power could not be turned on because the interlock signal is still there.
 You can use the **Diagnostics** table to toggle the motor/communication power to check the circuit of interlock at that time if you could not enable the system successfully.
 See the :ref:`lsst.ts.m2gui-user_diagnostics` for more details.
 
+If everything looks good, you can transition the system into the **Enabled** state to command the M2 hardware with the open-loop or closed-loop control (do not forget to turn off the motor and communication power first if this will be the first time to transition to the **Enabled** state).
+If the interlock is on, you would not be able to transition to the **Enabled** state.
+Sometimes, you may need the global interlock system (GIS) to reset the interlock signal of M2.
```
Comment (gabrirod86) [inline doc/user-guide/user-guide.rst:109]: add: By pushing the RESET blue button of the M2 ACTUATOR tab on the GIS cabinet at the level 2

**Review by gabrirod86 — APPROVED** [review]
Thanks!

File: doc/error-handling/error-handling.rst (hunk 1 above)
Comment (teweitsai) [reply doc/error-handling/error-handling.rst]: Added in the commit: [2fbe0cc](https://github.com/lsst-ts/ts_m2gui/pull/55/commits/2fbe0ccb8c2407391d367a675d2b3a2c410b8fb1). Thanks!

File: doc/user-guide/user-guide.rst (hunk 2 above)
Comment (teweitsai) [reply doc/user-guide/user-guide.rst:109]: Added in the commit: [2fbe0cc](https://github.com/lsst-ts/ts_m2gui/pull/55/commits/2fbe0ccb8c2407391d367a675d2b3a2c410b8fb1). Thanks!

**Pull request merged**